src/v/redpanda/tests/fixture.h

```diff
@@ -431,6 +462,17 @@ class redpanda_thread_fixture {
             config.get("cloud_storage_disable_archiver_manager")
               .set_value(legacy_upload_mode_enabled);
             config.get("iceberg_enabled").set_value(iceberg_enabled);
+
+            if (development_enable_cloud_topics) {
+                const auto time_since_epoch
+                  = std::chrono::system_clock::now().time_since_epoch();
+                config
+                  .get("enable_developmental_unrecoverable_data_corrupting_"
+                       "features")
+                  .set_value(ssx::sformat("{}", time_since_epoch));
```

Comment:
What is this even formatted as? 🤔 Missing cast to duration::seconds and `.count()` call?